src/components/admin/license-settings.js

```diff
@@ -5,18 +5,20 @@ import { useTranslation } from 'react-i18next';
 import { validateUrl } from '../../ui/validation.js';
 import { settingsLicenseShape } from '../../ui/default-prop-types.js';
 
+const urlValidationOptions = { allowEmpty: true, allowHttp: true };
+
 function LicenseSettings({ license, onChange }) {
   const { t } = useTranslation();
 
   const name = license?.name || '';
   const url = license?.url || '';
 
   const isUrlValid = newUrl => {
-    return validateUrl(newUrl, t, { allowEmpty: true, allowHttp: true });
+    return validateUrl(newUrl, t, urlValidationOptions).validateStatus !== 'error';
```

Comment:
This actually gets changed with this cleanup here https://github.com/educandu/educandu/pull/807/files 😋

Reply:
Removed :-)

Reply:
👌